src/components/Routes.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import Home from "./Home";
+import Err404 from "./Err404";
+import Contact from "./Contact";
+import About from "./About";
+import DailyHoroscope from "./DailyHoroscope";
+
+function Routes() {
+  const horoscopeSigns = [
```

Comment:
You should not repeat stuff when programming rather you should structure your data.
If you look in the file you will see that rather than writing manually path and props I structured the data on an array and therefore cascade that data on where needed.

You should read about data structures and DRY.